frontend/src/constants/fileTypes.js

```diff
@@ -1,39 +1,40 @@
 export const EXCEL = {
-  "application/vnd.ms-excel": ".xls",
-  "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet": ".xlsx",
+  ".xls": "application/vnd.ms-excel",
+  ".xlsx": "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet",
 };
+
 export const PDF = {
-  "application/pdf": ".pdf",
+  ".pdf": "application/pdf",
 };
+
 export const DOC = {
-  "application/msword": ".doc",
-  "application/vnd.openxmlformats-officedocument.wordprocessingml.document": ".docx",
-  "application/rtf": ".rtf",
-  "text/rtf": null,
```

Comment:
As I would understand this, files without an extension would be treated as rich text files. I don't think we have a requirement to deal with this so am alright to remove it.